variables.tf

```diff
@@ -341,8 +341,14 @@ variable "iam_roles" {
   type        = list(string)
   description = "Iam roles for the Aurora cluster"
   default     = []
+}
 
+variable "backtrack_window" {
+  type        = string
+  description = "The target backtrack window, in seconds. Only available for aurora engine currently. Must be between 0 and 259200 (72 hours)"
+  default     = "0"
 }
 
```

Comment:
remove empty lines and run `terraform fmt`

Reply:
done